modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequestStatus.testcase

```diff
@@ -0,0 +1,218 @@
+			EVPUtils.gotoEVP();
+
+			EVPOrganization.goToCreateOrganization();
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Bank Test",
+				city = "City Test",
+				contactEmail = "test@liferay.com",
+				contactName = "Contact Test",
+				contactPhone = 12345,
+				country = "Country Test",
+				createRequest = "true",
+				organizationName = "Organization Test",
+				organizationSite = "www.test.com.br",
+				smallDescription = "Description Test",
+				state = "State Test",
+				street = "Street Test",
+				taxId = 12345,
+				zipCode = "12.345-67");
+
+			EVPRequest.fillRequestForm(
+				grantAmount = 20,
+				grantRequestType = "Proposed",
+				organization = "Organization Test",
+				phoneNumber = 12345,
+				requestBehalf = "Global/Local Poor",
+				requestDescription = "Request Description Test",
+				requestPurposes = "Basic Education",
+				requestType = "Grant");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "EVP Portal");
+		}
+	}
+
+	@description = "LPS-181773 - Verify that the request status change to 'Awaiting Payment Confirmation' after clicking in 'Approve'"
+	@priority = 4
+	test CanFinanceUserChangeStatusAfterApprove {
+		task ("Given that the Manager user go to request page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "Manager User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Request");
+		}
+
+		task ("And the manager user approve the request created") {
+			EVPRequest.goToRequestOption(
+				key_option = "Review",
+				organizationName = "Organization Test");
+
+			EVPOrganization.changeStatus(
+				button = "Approve",
+				typeMessage = "Your request has been approved!");
+		}
+
+		task ("And after logging with finance user, he goes to request page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "finance@liferay.com",
+				userLoginFullName = "Finance User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Request");
+		}
+
+		task ("When the finance user approve the request created") {
+			EVPRequest.goToRequestOption(
+				key_option = "Review",
+				organizationName = "Organization Test");
+
+			EVPOrganization.changeStatus(
+				button = "Approve",
+				typeMessage = "Your request has been approved!");
+		}
+
+		task ("Then the status will be changed to 'Awaiting Payment Confirmation'") {
+			EVPUtils.viewStatus(
+				organizationName = "Organization Test",
+				status = "Awaiting Payment Confirmation");
+		}
+	}
+
+	@description = "LPS-181773 - Verify that the request status change to 'Awaiting More Info From Employee' after clicking in 'Request More Info'"
+	@priority = 4
+	test CanManagerUserChangeStatusAfterNeedMoreInfo {
```

Comment:
could you please check with Heidi about this "request more info" flow? I`m having exactly this form her part and we need  guarantee that we will not have two different code doing the same thing